libs/datasets/vaccine_backfills.py

```diff
@@ -96,16 +103,29 @@ def estimate_initiated_from_state_ratio(ds_in: MultiRegionDataset) -> MultiRegio
     assert ts_state_level_ratios.index.names == [STATE_LOCATION_ID]
     assert ts_state_level_ratios.columns.names == [CommonFields.DATE]
 
-    # Find counties that have vaccinations completed but not initiated. These are the only
-    # counties that will be modified.
+    # Find counties that have vaccinations completed but not initiated in recent
+    # data. These are the only counties that will be modified.
     ds_counties = ds_in.get_subset(AggregationLevel.COUNTY)
     ts_counties_initiated = ds_counties.get_timeseries_not_bucketed_wide_dates(
         CommonFields.VACCINATIONS_INITIATED
     )
     ts_counties_completed = ds_counties.get_timeseries_not_bucketed_wide_dates(
         CommonFields.VACCINATIONS_COMPLETED
     )
-    counties_to_modify = ts_counties_completed.index.difference(ts_counties_initiated.index)
+    recent_start = (datetime.today() - timedelta(days=APPLY_BACKFILL_LOOKBACK_DAYS)).strftime(
+        "%Y-%m-%d"
+    )
```

Comment:
Hrm...  I'm not sure?  Dropping `.strftime()` doesn't work.  Wrapping them with `pd.to_datetime()` didn't work.  I'm probably missing something about how pandas / python datetimes interact, but I think I'll just keep this since it works (sorry). 😁